dts/bindings/pinctrl/renesas,smartbond-pinctrl.yaml

```diff
@@ -83,6 +83,8 @@ child-binding:
         property-allowlist:
           - bias-pull-down
           - bias-pull-up
+          - output-enable
+          - input-enable
```

Comment:
pinctrl updates should better be moved to separate commit

Reply:
Done